Backend/app/therapy_sessions/views.py

```
@@ -1,3 +1,1541 @@
+                "is_online": True,
+                "patient_goals": "Emergency session for anxiety",
+                "consent_recording": True,
+                "consent_ai_analysis": True
+            },
+            request_only=True,
+        ),
+    ]
+)
+class TherapistSessionsView(generics.CreateAPIView):
+    """Create a new therapy session (therapists only)"""
+    permission_classes = [permissions.IsAuthenticated]
```

Ruff (0.12.2)

Redefinition of unused `permission_classes` from line 6

(F811)